tests/Seeds.ts

```diff
@@ -457,7 +457,6 @@ async function seed(): Promise<void> {
   });
   const MERCH_ITEM_1_PHOTO_1 = MerchFactory.fakePhoto({
     merchItem: MERCH_ITEM_1,
```

Comment:
To be honest, the link before was just a dummy link, and since I added the default dummy link to the MerchFactory, I think it is no longer necessary to have it here. Do you think it is better to declare it explicitly instead?

Reply:
As long as there's no issues, I think it's fine to have it in the MerchFactory since it's a generic property of all merch item photos. If we do this though, maybe we should consider abstracting away any other common properties for seed data that we create. not high priority but something to keep in mind